fineract-provider/src/test/java/org/apache/fineract/infrastructure/instancetype/DefaultInstanceTypeTest.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.fineract.infrastructure.instancetype;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import io.cucumber.java8.En;
+import org.apache.fineract.infrastructure.configuration.data.InstanceTypeData;
+import org.apache.fineract.infrastructure.configuration.service.InstanceTypeService;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class DefaultInstanceTypeTest implements En {
```

Comment:
I might be missing something here, but aren't the "*.feature" files missing? Please also see comments on properties mapping (probably less tests needed).